python/entrypoint.sh

```diff
@@ -396,6 +396,7 @@ if [[ -z "${AMBASSADOR_NO_KUBEWATCH}" ]]; then
            ${KUBEWATCH_SYNC_KINDS} \
            ${AMBASSADOR_FIELD_SELECTOR_ARG} \
            ${AMBASSADOR_LABEL_SELECTOR_ARG} \
+           --interval ${AMBASSADOR_WATCH_INTERVAL_MS:-250}ms \
```

Comment:
Taking a deeper look at this I don't see any tests for watt, that test this interval does what it's supposed too. If we're going to add it in this entrypoint, we should really test this.